templates/analytics/stats.html

```diff
@@ -7,7 +7,7 @@
 {% endblock %}
 
 {% block customhead %}
-{% stylesheet 'portico' %}
+{{ render_bundle('portico-styles') }}
```

Comment:
It's actually an unrelated bug that we have this stylesheet included directly in so many pages.  I've fixed that by merging 
697fd3c69b1fc8923097f3f152bbd980a44d844d; if you rebase this, you should be able to drop the changes in all but a couple files (since those no longer directly include the portico stylesheet).

Reply:
Rebased